### DISCUSSION TITLE
Simulate a water storage tank?

### DISCUSSION THREAD

**Discussion by RyanConway91** [modflowpy/flopy#2067] [Q&A]
I would like to simulate a water storage tank that is disconnected from the GW system (basically a water tower). Could I use the LAK package for this? I am thinking I could connect it to a single cell with a very low lake bed conductance and set up a stage/area/volume that makes sense for the tank. 

Anyone have a better idea or see any issues this approach?

**Comment by rhugman** [comment]
Modflow6 api. We did something along these lines for a yet-to-be published GMDSI worked example.

**Reply by RyanConway91** [reply]
Interesting, I'll look into using the API. I look forward to that worked example
